frontend/src/metabase/ui/components/inputs/Checkbox/Checkbox.styled.tsx

```diff
@@ -97,7 +97,7 @@ export const getCheckboxOverrides = (): MantineThemeOverride["components"] => ({
             width: "100%",
             borderRadius: rem(4),
             zIndex: -1,
-            backgroundColor: theme.white,
+            backgroundColor: theme.fn.themeColor("bg-white"),
```

Comment:
`theme.white` is not themeable by the SDK, while `bg-white` can be themed by `theme.colors.background`